12_TriggerImp.sql

```diff
@@ -0,0 +1,37 @@
+IF NOT EXISTS (
+    SELECT * FROM sys.tables t 
+    JOIN sys.schemas s ON (t.schema_id = s.schema_id) 
+    WHERE s.name = 'dbo' AND t.name = 'AuditLog') 
+    CREATE TABLE AuditLog (
+        LogID INT PRIMARY KEY IDENTITY(1,1),
+        BookID INT,
+        StatusChangeTo VARCHAR(50),
+		ChangeDate DATETIME
+    );
+GO
+CREATE OR ALTER TRIGGER trg_LogBooksStatusChanges
+ON Books
+AFTER UPDATE 
+AS
+BEGIN
+    INSERT INTO AuditLog (BookID, StatusChangeTo, ChangeDate)
+    SELECT 
+        BookID,
+        CASE
+			WHEN  CurrentStatus  = 'Available' THEN 'Available'
```

Comment:
😱  
it's as saying =>

```
if flag == true   return true
else if flag = false return  false
```
but it can simply be return flag.
and here we can just set it to CurrentStatus 😂

Reply:
Oops! 😅 I initially misunderstood the functionality, thinking it would be stored like this:` WHEN CurrentStatus = 'Available' THEN 'Borrowed' ...`. After checking, I realized it's not working like this; I edit 'Borrowed' to 'Available.'  without forcusing on it 😅 Sorry for that.